packages/react/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../configs/ts/tsconfig.react.json",
   "compilerOptions": {
     "noImplicitAny": false,
-    "strict": false
+    "strict": true
```

Comment:
You can just delete this line actually, `tsconfig.react.json` has `strict` set to `true`